Readme.md

```diff
@@ -81,29 +83,23 @@ Experimental feature - requires improvements.
 
 Use `preloadResources: true` to enable this feature.
 
-## Recipes
-
-See [recipes](Recipes.md) for more examples.
-
-## Caveats
```

Comment:
Inline css and Preload resources this is advanced features. Which you chose to enable or not.

But everything else is in the section `Caveats` (also can be called troubleshooting), because you want to touch this only if you have issues.

Google Analytics, Mapbox, WebGL are not a configs of `react-snap` but rather technologies which can cause troubles.

Reply:
OK, now I think this section doesn't need to change, the document is enough for people to know how to add options, and there is no need to put all options together. Closing this.